src/widget/wknobcomposed.cpp

```diff
@@ -116,6 +114,26 @@ void WKnobComposed::onConnectedControlChanged(double dParameter, double dValue)
     }
 }
 
+void WKnobComposed::setDefaultAngleFromParameterOrReset(std::optional<double> parameter) {
+    if (!parameter.has_value() || parameter.value() < 0 || parameter.value() > 1) {
+        m_defaultAngle = std::nullopt;
+    } else {
+        m_defaultAngle = std::lerp(m_dMinAngle, m_dMaxAngle, parameter.value());
```

Comment:
you may try 
```suggestion
    if (!parameter.has_value() || *parameter < 0 || *parameter > 1) {
        m_defaultAngle = std::nullopt;
    } else {
        m_defaultAngle = std::lerp(m_dMinAngle, m_dMaxAngle, *parameter);
```

Reply:
okay, great, that did the trick!
and I added a TODO as @Swiftb0y suggested.